chia/simulator/setup_services.py

```diff
@@ -98,9 +98,10 @@ async def setup_full_node(
     connect_to_daemon: bool = False,
     db_version: int = 1,
     disable_capabilities: Optional[List[Capability]] = None,
+    reuse_db: bool = False,
```

Comment:
I would have preferred adding `*,` here, to force this bool to be named. but it's not a big deal

Reply:
```suggestion
    *,
    reuse_db: bool = False,
```